ports/esp32/boards/ARDUINO_NANO_ESP32/modules/arduino_pins.py

```diff
@@ -0,0 +1,76 @@
+#
```

Comment:
It looks like this module is intended to work around the fact that the esp32 port doesn't have named pins, is that correct?

Maybe it's better to implement a very simple named pins feature in esp32 instead?

@jimmo what do you think?

Reply:
I'm implementing named pins on esp32, so this can become a pins.csv file.  That will be better.

Reply:
Sounds great, thanks! One question, do I need to add this board to auto-build scripts ?

Reply:
> I'm implementing named pins on esp32, so this can become a pins.csv file. That will be better.

I was about to ask whether I should do that, similar to other ports like the SAMD. When doing so, please have a look at PR #11731 and the changes to the ESP32 port, which unifies the way pins are specified within the port, and across the other ports of the PR.
With ESP32, pins have sometimes to be specified as just a number, sometimes as Pin(x).

Reply:
> do I need to add this board to auto-build scripts

If it has a board.json then it will be automatically built and available on the download page.

Reply:
See #12010 for named pins on esp32.

Reply:
I rebased this on #12010 and converted the pins module to `pins.csv` and tested it, it seems to be working fine.

Reply:
Will you remove this .py file then?

Reply:
removed.